lib/rules/declaration-block-trailing-semicolon/README.md

```diff
@@ -88,3 +88,38 @@ a { color: pink }
 ```css
 a { background: orange; color: pink }
 ```
+
+## Optional secondary options
+
+### `ignore: ["single-declaration"]`
+
+#### `"single-declaration"`
+
+The following patterns are _not_ considered violations:
+
+<!-- prettier-ignore -->
+```css
+foo { property: value }
+```
+
+<!-- prettier-ignore -->
+```css
+foo { property: value; }
+```
+
+<!-- prettier-ignore -->
+```css
+@keyframes name { from { property: 0 } to { property: 1; } }
+```
+
```

Comment:
```suggestion
```

Let's remove this example to be consistent with other rules where we only show the simplest example of the option in effect.